packages/spruce-next-helpers/src/skillskit/index.ts

```diff
@@ -276,8 +494,36 @@ const skill = {
 		return modal
 	},
 
+	calendar: () => {
+		const calendar: ICalendar = {
+			createEvent: event => {
+				Iframes.sendMessage({
+					to: window.parent,
```

Comment:
Weren't you saying that there was an iOS consideration in the other PR? Will we need to add another method call for each event in this file?